client-js/src/client/backend-client.js

```diff
@@ -153,39 +246,22 @@ export class BackendClient {
   }
```

Comment:
- Is `BackendClient` an entity that is applicable to any implementation of `BackendClient`, or is it Firebase-specific? (BTW, I'd make it clear in its JSDocs, since we have an abstract `BackendClient` with its specific implementation snow).
- If it is not Firebase-specific, is this imlementation logic of `fetchAll` Firebase-specific?
- If no, should we actually move it to `BackendClient` so that this method is not abstract there?
- The same questions apply to the other methods in this class.

Reply:
After a face-to-face discussion, I’ve separated `Fetch` and `BackendClient` abstract definition and implementation further.